examples/test/post-processing/components/compositor.js

```diff
@@ -0,0 +1,196 @@
+/* global AFRAME THREE */
+
+AFRAME.registerComponent('post-process', {
+  schema: {type: 'boolean'},
+
+  update: function (oldData) {
+    this.el.sceneEl.enablePostProcessing = this.data;
+  },
+
+  // Cleanup the scene objects opacities.
+  remove: function () {
+    this.el.sceneEl.object3D.traverse(function (obj) {
+      var mat = obj.material;
+      if (mat && mat.postOpacity !== undefined) {
+        mat.opacity = 1;
+      }
+    });
+    this.el.sceneEl.enablePostProcessing = false;
+  }
+});
+
+// The compositor. Blends the blurred renderTarget from the bloom component.
+// It also checks opacity and converts parts of the frame to ascii(Alpha masking).
+AFRAME.registerSystem('post-process', {
+  init: function () {
+    var self = this;
+    this.setupPostState();
+    this.postModified();
+    this.sceneEl.addEventListener('renderstart', function () {
+      self.sceneEl.renderTarget.depthTexture = new THREE.DepthTexture();
+      self.sceneEl.renderTarget.stencilBuffer = false;
+    });
+    this.evh = this.postModified.bind(this);
+    this.sceneEl.addEventListener('post-modified', this.evh);
+  },
+
+  setupPostState: function () {
+    this.scenePost = new THREE.Scene();
+    this.cameraPost = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1);
+    this.quadPost = new THREE.Mesh(new THREE.PlaneBufferGeometry(2, 2), null);
+    this.scenePost.add(this.quadPost);
+  },
+
+  postModified: function (ev) {
+    this.needsConfig = true;
+  },
+
+  // Whenever the postproc setup changes, the compositor shader is recompiled
+  config: function () {
+    var scene = this.sceneEl;
+    var behaviors = scene.behaviors.tock;
+    var bloom = behaviors.length ? behaviors[0] : null;
+    console.log('POST-MODIFIED', behaviors);
```

Comment:
not really, I put the logs in just so there can be a reference of things that could be useful for anyone writing post proc. Should I take them out?